### PR TITLE
[MM-32692] Add two more CRT options

### PR DESCRIPTION
**Pull request by vish9812**
#### Summary
Feature: Add two more CRT options in System Console:  default_on & always_on

#### Ticket Link
JIRA: https://mattermost.atlassian.net/browse/MM-32692

#### Release Note
```release-note
Enabled always-on and default-on for CRT system configuration
```

### CODE REVIEW

File: app/channel_test.go [hunk 1]
```diff
@@ -2447,6 +2447,87 @@ func TestMarkUnreadWithThreads(t *testing.T) {
...
+		},
+		{
+			desc:     "Returns false when system config is default_off and user has off preference",
+			appCRT:   model.CollapsedThreadsDefaultOff,
+			pref:     preference{"off", nil},
+			expected: false,
+		},
+	}
+
+	for _, tC := range testCases {
```
Comment (ashishbhate) [inline app/channel_test.go:2510]: I think we should name this `tc` or `testCase` instead of `tC`

**Review by ashishbhate — APPROVED** [review]

File: app/channel_test.go (hunk 1 above)
Comment (vish9812) [reply app/channel_test.go]: Done.

**Review by koox00 — APPROVED** [review]

**Comment by jgilliam17** [comment]
@vish9812 Can you check why the ee/circleci check is pending. We need all green checks for E2Es to start. Thanks

**Review by jgilliam17 — APPROVED** [review]
Tested on related webapp PR test server. Looks good to merge.
E2E [report](https://mattermost-cypress-report.s3.amazonaws.com/192959-2cdc128-onprem-ent-server-pr-20088-mattermost/mm-ee-test2cdc128/mochawesome.html) ✅ , no PR related failures.

**Pull request merged**